kitsune/wiki/views.py

```diff
@@ -125,6 +125,32 @@ def document(request, document_slug, template=None, document=None):
             # and OK to fall back to parent (parent is approved).
             fallback_reason = 'no_translation'
 
+    # If any local has fallback locals and the docmunet is missing localization in that local
+    # So show the fallback local's document. There can be highest two fallback local for any local
+    # Fallback locals can be add in kitsune/wiki/config.py
+    if fallback_reason == 'no_translation' and (request.LANGUAGE_CODE in FALLBACK_LOCALS.keys()):
+        # Get fallback locals for the requested local
+        fallback_local_code = FALLBACK_LOCALS.get(request.LANGUAGE_CODE)
```

Comment:
This should probably be written as `FALLBACK_LOCALS[request.LANGUAGE_CODE]`. `.get()` is mainly used in case the key might not exist in the dictionary. You already know that the language code is in `FALLBACL_LOCALS`, thanks to the `if` statement above.